pytissueoptics/scene/geometry/bbox.py

```diff
@@ -22,6 +22,15 @@ def __eq__(self, other: 'BoundingBox') -> bool:
         else:
             return False
 
+    def __format__(self, formatSpec):
+        f_xMin = float(format(self.xMin, formatSpec))
+        f_xMax = float(format(self.xMax, formatSpec))
+        f_yMin = float(format(self.yMin, formatSpec))
+        f_yMax = float(format(self.yMax, formatSpec))
+        f_zMin = float(format(self.zMin, formatSpec))
+        f_zMax = float(format(self.zMax, formatSpec))
+        return str([[f_xMin, f_xMax], [f_yMin, f_yMax], [f_zMin, f_zMax]])
+
```

Comment:
It is analog to the `__repr__`, but it allows to use format strings, in instance when bbox is printed, but one would want to round to the 2nd decimal f"{object.bbox:.2f}" would call `__format__` instead of `__repr__`.